### DISCUSSION TITLE
importing .raw file into the jupyter notebook

### DISCUSSION THREAD

**Discussion by JahanzebTariq** [PMEAL/OpenPNM#2638] [Q&A]
Dear Professor.. could you please tell me how to load .raw file though this code.. or is there a better way to do it.  I took this code fraom a script but was not able to make it work.. please guide me

  # %% Read image
    path = Path(os.path.realpath(__file__),
                '../../../tests/fixtures/berea_100_to_300.npz')
    data = np.load(path.resolve())
    im = data['im']

**Comment by jgostick** [comment]
To get a '.raw' file into Jupyter you might be able to use ``imageio``, but you're probably best off using good old ImageJ to open it then save it as standard tiff image, which you can then open with ``imageio`` in Jupyter.
